internal/validator/validate.go

```diff
@@ -0,0 +1,61 @@
+package validator
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/docker/app/internal"
+	"github.com/docker/app/internal/render"
+	"github.com/docker/app/internal/templateloader"
+	"github.com/docker/app/specification"
+)
+
+const (
+	// Current metadata version
+	metadataVersion = "v0.1"
```

Comment:
I would abstract this out to something like `internal/config.go` so that we have all our configuration in one place.

Reply:
Maybe in a follow up with others constants (like the templating formats supported)?

Reply:
Oh right it already exists... I'll update it.